subsys/mgmt/mcumgr/Kconfig

```diff
@@ -75,6 +309,7 @@ config MCUMGR_SMP_UDP
 	  Enables handling of SMP commands received over UDP.
 	  Will start a thread for listening on the configured UDP port.
 
+if MCUMGR_SMP_UDP
```

Comment:
Please remove `depends on MCUMGR_SMP_UDP` in all options between `if MCUMGR_SMP_UDP` and `endif # MCUMGR_SMP_UDP`

Reply:
Done, thanks!